crates/goose-cli/src/session/mod.rs

```diff
@@ -179,6 +217,42 @@ impl Session {
                     continue;
                 }
                 input::InputResult::Retry => continue,
+                input::InputResult::ListPrompts => {
+                    output::render_prompts(&self.list_prompts().await)
+                }
+                input::InputResult::PromptCommand(opts) => {
+                    // name is required
+                    if opts.name.is_empty() {
+                        output::render_error("Prompt name argument is required");
+                        continue;
+                    }
+
+                    if opts.info {
+                        match self.get_prompt_info(&opts.name).await? {
+                            Some(info) => output::render_prompt_info(&info),
+                            None => {
+                                output::render_error(&format!("Prompt '{}' not found", opts.name))
+                            }
+                        }
+                    } else {
+                        // Convert the arguments HashMap to a Value
+                        let arguments = serde_json::to_value(opts.arguments)
+                            .map_err(|e| anyhow::anyhow!("Failed to serialize arguments: {}", e))?;
+
+                        match self.get_prompt(&opts.name, arguments).await {
+                            Ok(messages) => {
```

Comment:
do you think its useful to render the expanded prompt here?

Reply:
oh yeah, we should - the user might be confused if its just `/prompt ...` and things start happening without showing the expanded prompt.